.github/scripts/cleanup_pr_body.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
```

Comment:
Might be worth it to add `set -eu` so that the workflow fails on any errors and/or unset variables?